fxapom/fxapom.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import string
+import random
+from datetime import datetime
+
+from fxa.core import Client
+from fxa.tests.utils import TestEmailAccount
+
+
+class WebDriverFxA(object):
+
+    def __init__(self, testsetup):
+        self.testsetup = testsetup
+
+    def sign_in(self, email=None, password=None):
+        """Signs in a user, either with the specified email address and password, or a returning user."""
+        from pages.sign_in import SignIn
+        sign_in = SignIn(self.testsetup)
+        sign_in.sign_in(email, password)
+
+
+class FxATestAccount:
+    """A base test class that can be extended by other tests to include utility methods."""
+
+    password = ''.join([random.choice(string.letters) for i in xrange(8)])
```

Comment:
I'm not sure. This was in the code when I started making my mods. From what I can understand, `xrange` is potentially faster, but that shouldn't matter in this case. I also understand that `xrange` is not Python 3 compatible, so I will change it to `range`.